src/components/Portal/PortalInner.tsx

```diff
@@ -0,0 +1,48 @@
+import * as PropTypes from 'prop-types'
+import { invoke } from 'lodash'
```

Comment:
Let's keep convention consistent in the repo, it makes tooling and programmatic updates easier to manage.

There is a `babel-plugin-lodash` which we use at SUIR to rewrite the import statements to cherry pick only used methods.  We should either implement this plugin, or enforce cherry picking lodash methods, but, we should not mix both patterns without a guarantee of either one.